test/test_helper.exs

```diff
@@ -63,9 +67,26 @@ defmodule TestHelpers do
       next_player: human_player2()
     }
 
-  def mock_board(cell_nums, marker, size) do
-    board = Enum.to_list(1..size)
+  def mock_board(moves, marker, size) do
+    Enum.to_list(1..size)
+    |> add_moves(moves, marker)
+  end
+
+  def mock_board(options) do
+    %{
+      player1_marker: player1_marker,
+      player1_moves: player1_moves,
+      player2_marker: player2_marker,
+      player2_moves: player2_moves,
+      size: size
+    } = Enum.into(options, %{})
```

Comment:
Non-blocking FYI, you can combine these two. Since `options` is a keyword list, you can write:

```elixir
def mock_board(
  player1_marker: player1_marker,
  player1_moves: player1_moves,
  player2_marker: player2_marker,
  player2_moves: player2_moves,
  size: size
) do
  # stuff
end
```

Since the keyword list is the last argument (or the only argument in this case), you can omit the `[]`s